snuba/cli/subscriptions.py

```diff
@@ -119,6 +119,7 @@ def subscriptions(
     dataset = get_dataset(dataset_name)
 
     storage = dataset.get_default_entity().get_writable_storage()
+    assert storage is not None
```

Comment:
Please provide a meaningful error message as the user will see this and won't know why their input was accepted but subscriptions are not working.